tests/components/tuya/snapshots/test_switch.ambr

```diff
@@ -5704,6 +5704,54 @@
     'state': 'off',
   })
 # ---
+# name: test_platform_setup_and_discovery[switch.kattenbak_none-entry]
```

Comment:
`kattenbak_none` => I think you didn't recompile the translations before updating the snapshot